### ISSUE TITLE
Without login, chromium opens to tweet from 2015

### ISSUE THREAD

**Issue by xelab04** [Danny-Nunez/python-twitter-scraper#2]
Hello, really sorry to bother. This software is great and works really well, thank you very much!

Only issue I am having is that when I run the code, it ends up with a post from 2015. Only after login, does the web interface show the latest tweets.
Do you think it would be possible to automate a login in order to bypass this?

**Comment by xelab04** [comment]
I am bypassing this manually by enabling the chromium window. After opening the window, it hangs on an "input" command which pauses execution to login manually. Then it can continue and get the latest tweet

**Comment by Danny-Nunez** [comment]
Yes, you can automate login inputs. The most challenging part would be configuring your server to run Chromium for this purpose. Instead, I recommend focusing on inspecting the landing page to identify a specific class or element that can serve as a trigger for scraping your Twitter account.
